site/content/docs/design/base-image.md

```diff
@@ -12,9 +12,16 @@ menu:
 The ["base" image][base image] is a small-ish Docker image for running
 nested containers, systemd, and kubernetes components.
 
-To do this we need to set up an environment that will meet the CRI 
-(currently just docker) and systemd's particular needs. Documentation for each
-step we take is inline to the image's [Dockerfile][dockerfile],
+Logic for building ["base" image][base image] can be found in
```

Comment:
This is not true anymore.

Reply:
There's a makefile, IIRC the quick start page references this.

Reply:
I'll get this updated, then, in a bit.

Reply:
https://github.com/kubernetes-sigs/kind/pull/1568/commits/538f0b9b3d0a0c18b263632a2bdea434c3bccd77